sktools/src/sktools/scripts/skexp.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
+
+'''Returns STO exponents according to the usual geometric series.'''
```

Comment:
Might be worth adding a reference, especially if the 0.5 default comes from somewhere. Also returns or prints out?

Reply:
Works for me, do you have a reference at hand?

"Prints" is more precise at the moment, I can change the wording.

Reply:
The 0.5 starting point was just an arbitrary value that I remembered and to some extend a convention established by the www.dftb.org repo.

Reply:
Thanks Ben, at least it contains the idea of the geometric series, I will just add the reference accordingly.